Aztec/Classes/TextKit/TextView.swift

```diff
@@ -281,17 +285,69 @@ open class TextView: UITextView {
 
     // MARK: - Apparance Properties
 
+    /// Blockquote max depth, enter 1 - 5, default is 5
+    @objc dynamic public var blockquoteMaxDepth: Int {
+        get {
+            return layout.quoteMaxDepth + 1
+        }
+        set {
+            if newValue < 0 {
+                layout.quoteMaxDepth = 0
+            } else if newValue <= 5 {
+                layout.quoteMaxDepth = newValue - 1
+            } else {
+                layout.quoteMaxDepth = 4
+            }
+        }
+    }
+    
     /// Blockquote Blocks Border Color.
     ///
-    @objc dynamic public var blockquoteBorderColor: UIColor? {
+    @objc dynamic public var blockquoteBorderColor: UIColor {
         get {
-            return layout.blockquoteBorderColor
+            return layout.blockquoteBorderColor[0]!
         }
         set {
-            layout.blockquoteBorderColor = newValue
+            layout.blockquoteBorderColor[0] = newValue
         }
     }
 
+    @objc dynamic public var blockquoteBorderColorNested1: UIColor {
```

Comment:
I think it will probably be easier and more flexible to expose the full color array for quotes in layout, instead of having this "shortcut" for each level.

Reply:
Hmm, I guess this would work. Originally I wanted to keep the colors limited to max of 5 and I wasn't sure how to limit the length of the array a user might submit. However, if thats not a concern, I guess it would work to just let them submit any array they choose?